docs/web3.eth.account.rst

```diff
@@ -0,0 +1,362 @@
+        
+        # variations on msghash
+        >>> msghash = b"\x14v\xab\xb7E\xd4#\xbf\t'?\x1a\xfd\x88}\x95\x11\x81\xd2Z\xdcf\xc4\x83JpI\x19\x11\xb7\xf7P"
+        >>> w3.eth.account.recover(msghash, vrs=vrs)
+        >>> msghash = 0x1476abb745d423bf09273f1afd887d951181d25adc66c4834a70491911b7f750
+        >>> w3.eth.account.recover(msghash, vrs=vrs)
+
+        # variations on vrs
+        >>> vrs = (
+              '0x1c',
+              '0xe6ca9bba58c88611fad66a6ce8f996908195593807c4b38bd528d2cff09d4eb3',
+              '0x3e5bfbbf4d3e39b1a2fd816a7680c19ebebaf3a141b239934ad43cb33fcec8ce')
+        >>> w3.eth.account.recover(msghash, vrs=vrs)
+        >>> vrs = (
+              b'\x1c',
+              b'\xe6\xca\x9b\xbaX\xc8\x86\x11\xfa\xd6jl\xe8\xf9\x96\x90\x81\x95Y8\x07\xc4\xb3\x8b\xd5(\xd2\xcf\xf0\x9dN\xb3',
+              b'>[\xfb\xbfM>9\xb1\xa2\xfd\x81jv\x80\xc1\x9e\xbe\xba\xf3\xa1A\xb29\x93J\xd4<\xb3?\xce\xc8\xce')
+        >>> w3.eth.account.recover(msghash, vrs=vrs)
+        >>> vrs = (
+              0x1c,
+              0xe6ca9bba58c88611fad66a6ce8f996908195593807c4b38bd528d2cff09d4eb3,
+              0x3e5bfbbf4d3e39b1a2fd816a7680c19ebebaf3a141b239934ad43cb33fcec8ce)
+        >>> w3.eth.account.recover(msghash, vrs=vrs)
+
+        # variations on signature
+        >>> signature = '0xe6ca9bba58c88611fad66a6ce8f996908195593807c4b38bd528d2cff09d4eb33e5bfbbf4d3e39b1a2fd816a7680c19ebebaf3a141b239934ad43cb33fcec8ce1c'
+        >>> w3.eth.account.recover(msghash, signature=signature)
+        >>> signature = b'\xe6\xca\x9b\xbaX\xc8\x86\x11\xfa\xd6jl\xe8\xf9\x96\x90\x81\x95Y8\x07\xc4\xb3\x8b\xd5(\xd2\xcf\xf0\x9dN\xb3>[\xfb\xbfM>9\xb1\xa2\xfd\x81jv\x80\xc1\x9e\xbe\xba\xf3\xa1A\xb29\x93J\xd4<\xb3?\xce\xc8\xce\x1c'
+        >>> w3.eth.account.recover(msghash, signature=signature)
+        >>> signature = 0xe6ca9bba58c88611fad66a6ce8f996908195593807c4b38bd528d2cff09d4eb33e5bfbbf4d3e39b1a2fd816a7680c19ebebaf3a141b239934ad43cb33fcec8ce1c
+        >>> w3.eth.account.recover(msghash, signature=signature)
+
+
+.. py:method:: Account.hashMessage(data=None, hexstr=None, text=None)
+
+    Generate the message hash, including the prefix. See :meth:`Account.sign`
+    for more about the prefix. Supply exactly one of the three arguments.
+
+    :param data: the message to sign, in primitive form
+    :type data: bytes or int
+    :param str hexstr: the message to sign, as a hex-encoded string
+    :param str data: the message to sign, as a series of unicode points
+    :returns str: the hex-encoded hash of the message
+
+    .. code-block:: python
+
+        >>> msg = "I♥SF"
+        >>> w3.eth.account.hashMessage(text=msg)
+        '0x1476abb745d423bf09273f1afd887d951181d25adc66c4834a70491911b7f750'
+
+
+.. py:method:: Account.signTransaction(transaction_dict, private_key)
+
+    Sign a transaction using a local private key. Produces signature details
+    and a rawTransaction that you can broadcast.
```

Comment:
Maybe better? 

*"and the hex encoded transaction suitable for broadcast using `web3.eth.sendRawTransaction`."*